wcs-functions.php

```diff
@@ -458,7 +458,7 @@ function wcs_get_subscriptions( $args ) {
 		'status'     => $args['subscription_status'],
 		'limit'      => $args['subscriptions_per_page'],
 		'page'       => $args['paged'],
-		'offset'     => $args['offset'],
+		'offset'     => $args['offset'] > 0 ? $args['offset'] : null,
```

Comment:
I discovered while testing this PR that there is a bug when `wcs_get_subscriptions()` is called with the `paged` arg. 

WC's order query (which this function will eventually call), will ignore the page arg if there's an offset. This is also documented [here](https://github.com/woocommerce/woocommerce/wiki/wc_get_orders-and-WC_Order_Query#:~:text=orders%20%3D%20wc_get_orders(%20%24args%20)%3B-,paged,-Accepts%20an%20integer). 

This includes if the offset is 0. 

You can test this by doing something like this: 

```php
$args = array(
	'customer_id'            => 1,
	'subscriptions_per_page' => 3,
	'paged'                  => 2,
	'return'                 => 'ids',
);

error_log( print_r( array_keys( wcs_get_subscriptions( $args ) ), true ) );
```

On `trunk` this will return results like this:

```
Array
(
    [0] => 9592
    [1] => 9589
    [2] => 9587
)
```

Change the `paged` arg to 3 or something else and rerun. You'll notice that you get the same results. ie paging is not applying. 

```
Array
(
    [0] => 9592
    [1] => 9589
    [2] => 9587
)
```

Checkout this branch and rerun. You should get different results.

eg
```
Array
(
    [0] => 9585
    [1] => 9583
    [2] => 9581
)
```